entity-server-api/src/main/java/org/terracotta/entity/ActiveServerEntity.java

```diff
@@ -50,20 +58,23 @@
   R invoke(ClientDescriptor clientDescriptor, M message);
 
   /**
-   * Called during client reconnect to allow the client to pass arbitrary extra data to the server-side entity so it can
-   * rebuild any in-memory state it had, related to the connected client.
-   * Note that this is called AFTER connected() is called for this clientDescriptor.
+   * <p>Called during client reconnect to allow the client to pass arbitrary extra data to the server-side entity so it can
+   *  rebuild any in-memory state it had, related to the connected client.</p>
+   * <p>Note that this is called AFTER connected() is called for this clientDescriptor.</p>
+   * <p>Note that this call is made on the {@link ConcurrencyStrategy#MANAGEMENT_KEY}, meaning that it is serialized with
+   *  respect to all other messages enqueued for the entity.</p>
    * 
    * @param clientDescriptor The client-side instance which reconnected
    * @param extendedReconnectData Arbitrary data sent by the client-side instance to rebuild the server-side in-memory state 
    */
   void handleReconnect(ClientDescriptor clientDescriptor, byte[] extendedReconnectData);
 
   /**
-   * Passes any information required to describe all entity data/state associated with the given concurrency key to a
-   * passive instance being synchronized to be consistent with the receiver.
+   * <p>Passes any information required to describe all entity data/state associated with the given concurrency key to a
+   *  passive instance being synchronized to be consistent with the receiver.</p>
    * 
-   * Note that this method is also run on the concurrencyKey specified, so it blocks other messages executed on that key.
+   * <p>Note that this method is also run on the concurrencyKey specified, so it blocks other messages executed on that
+   *  key.</p>
    * 
```

Comment:
Do we need to add a note here that the invocation of this method could happen concurrently on different concurrency keys?